packages/react-compose/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-compose-enhancers/README.md",
   "dependencies": {
-    "hoist-non-react-statics": "^2.5.0"
+    "hoist-non-react-statics": "^3.0.1"
```

Comment:
why was this dependency updated? Can that be done in a separate PR?

Reply:
It was updated because of some typing issues and the problems that come from having mismatched types in different sub-packages. IMO this is fine. Especially given how simple `hoist` is.